components/BillDetails.js

```diff
@@ -28,10 +33,7 @@ const Container = styled.div`
   padding: 1rem 2rem;
   flex-direction: column;
   border-radius: 1.5625em;
```

Comment:
Does this value really has much difference if you use `1.5em` or `1.6em` instead?